package.json

```diff
@@ -11,7 +11,9 @@
 	},
 	"license": "MIT",
 	"devDependencies": {
+		"@types/bootstrap": "^5.2.4",
 		"@types/jest": "^27.4.1",
+		"@types/jquery": "^3.5.14",
```

Comment:
Since #28 this dependency should now go.

Reply:
Done.